### PR TITLE
I have finished send email implementation

### PR DESCRIPTION
**Pull request by tiko-star**
Please review my PR.

### CODE REVIEW

File: send_email.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+
```
Comment (tiko-star) [inline send_email.php:3]: **Please add strict type**

File: send_email.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+mail('tiko@ucraft.com', 'Hello', 'Hello, how r u?');
```
Comment (tiko-star) [inline send_email.php]: return success message

File: send_email.php [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?php
```
Comment (AniKhachatryan333) [inline send_email.php:1]: hello

File: send_email.php [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+declare(strict_types = 1);
```
Comment (AniKhachatryan333) [inline send_email.php:2]: hii

**Review by AniKhachatryan333 — CHANGES_REQUESTED** [review]

**Review by AniKhachatryan333 — APPROVED** [review]

**Pull request merged**